src/Components/Milestones/MilestoneButton.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Modal from '@mui/material/Modal';
+import Box from '@mui/material/Box';
+
+const style = {
+    position: 'absolute',
```

Comment:
Can we move these styles into css file?

Reply:
For some reason, the inline does not work as intended when placed into the CSS file so at the moment we will leave it as is, but moving forward we will place the CSS in its respective file!